src/Widgets/DwellClickTimer.vala

```diff
@@ -100,6 +94,27 @@ namespace Gala {
         }
 
         public override void paint (Clutter.PaintContext context) {
+            /* We create a dummy Gtk label to get the stylesheet accent color*/
+            var dummy_label = new Gtk.Label ("");
+
+            unowned Gtk.StyleContext label_style_context = dummy_label.get_style_context ();
+
+            var widget_path = label_style_context.get_path ().copy ();
+            widget_path.iter_set_object_name (-1, "selection");
+
+            var style_context = new Gtk.StyleContext ();
+            style_context.set_path (widget_path);
+            style_context.set_parent (label_style_context);
+            style_context.set_state (Gtk.StateFlags.SELECTED);
+
+            var rgba = (Gdk.RGBA) style_context.get_property (
+                Gtk.STYLE_PROPERTY_BACKGROUND_COLOR,
+                Gtk.StateFlags.NORMAL
+            );
+
+            stroke_color = new Cairo.Pattern.rgb (rgba.red, rgba.green, rgba.blue);
+            fill_color = new Cairo.Pattern.rgba (rgba.red, rgba.green, rgba.blue, BACKGROUND_OPACITY);
```

Comment:
Just for clarity, why aren't you using rgba.alpha here instead of BACKGROUND_OPACITY ?

Reply:
Added a comment. It's just too low contrast in many situations. We can predict the background for a label, but not the cursor